scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -201,7 +201,8 @@
 {% macro variants_common_filters(form, type) %}
 <div class="row mb-2">
   <div class="col-2">
-    {{ form.gene_panels.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="left", title="This list can be modified from the institute settings page. Latest panel version is used in variants filtering.") }}
+    <span>{{ form.gene_panels.label(class="control-label", data_bs_toggle="tooltip", data_bs_placement="left", title="This list can be modified from the institute settings page. Latest panel version is used in variants filtering.") }}</span>
+    <span style="float:right;">{{ form.gene_panels_exclude.label(class="control-label") }} {{form.gene_panels_exclude}}</span>
```

Comment:
A hover with the details here perhaps? Or some other grouping to show that the option also affects the genes listed. It is easy to get confused about what happens when you have something in both "Gene Panels" and "HGNC Symbols" and tick "exclude panels". I do think you have chosen the best way to do it though: there is really no use-case for excluding panels and including genes at the same time.

Reply:
Yes, very good idea, also to explain that if the checkbox is checked it will be excluding from the search any gene provided in the form (gene panels, files with genes, or genes provided manually in the input box down)